pkgs/by-name/lo/loot/package.nix

```
@@ -0,0 +1,145 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, callPackage
+, boost
+, cmake
+, git
+, libloot
+, ogdf
+, pkg-config
+, qt6Packages
+, tbb_2021_11
+, tomlplusplus
+, xdg-utils
+, zlib
+}:
+
+let
+
+  GTest = fetchFromGitHub {
+    owner = "google";
+    repo = "googletest";
+    rev = "v1.14.0";
+    hash = "sha256-t0RchAHTJbuI5YW4uyBPykTvcjy90JW9AOPNjIhwh6U=";
+  };
+
+  # TODO use pkgs.minizip-ng (cmakeFlags = [ "-DMINIZIP_NG_LIBRARIES=${minizip-ng}/lib/libminizip-ng.so" ])
+  # /build/source/src/gui/backup.cpp:28:10: fatal error: mz.h: No such file or directory
+  minizip-ng = fetchFromGitHub {
+    owner = "zlib-ng";
+    repo = "minizip-ng";
+    rev = "4.0.7";
+    hash = "sha256-scoEqymRMBTZZVr1fxtKOyBj4VLCgI8jQpanUKrJhiQ=";
+  };
+
+  # TODO use pkgs.spdlog
+  spdlog = fetchFromGitHub {
+    owner = "gabime";
+    repo = "spdlog";
+    rev = "v1.14.1";
+    hash = "sha256-F7khXbMilbh5b+eKnzcB0fPPWQqUHqAYPWJb83OnUKQ=";
+  };
+
+  testing-plugins = callPackage ../../li/libloot/testing-plugins.nix { };
+
+  ValveFileVDF = fetchFromGitHub {
+    owner = "TinyTinni";
+    repo = "ValveFileVDF";
+    rev = "1a132f3b0b3cf501bdec03a99cdf009d99fc951c";
+    hash = "sha256-H5mbqLO1fJ0oa8d1Vl0KqcHgVyqwPR/kWCSrlSrKxSE=";
+  };
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "loot";
+  version = "0.23.0";
+
+  src = fetchFromGitHub {
+    owner = "loot";
+    repo = pname;
+    rev = version;
+    hash = "sha256-/jyAKslJBShwHhREEAi6YVty+Rfq6TO19U5gN9V2ljA=";
```

```suggestion
    hash = "sha256-/BoMSIHv0chS2AxhlY91kEQtSUCqLgAegYzwBPU/rKw=";
```
I would advise to patch the CMake file instead of `leaveDotGit = true`, it seems the fetch is not deterministic.